spring-cloud-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosDataPropertiesParser.java

```diff
@@ -16,23 +16,48 @@
 
 package com.alibaba.cloud.nacos.parser;
 
+import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.StringReader;
+import java.util.Arrays;
 import java.util.Properties;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 /**
  * @author zkz
+ * @author element
  */
 public class NacosDataPropertiesParser extends AbstractNacosDataParser {
 
+	private static final Logger log = LoggerFactory
+			.getLogger(NacosDataPropertiesParser.class);
+
 	public NacosDataPropertiesParser() {
 		super("properties");
 	}
 
 	@Override
 	protected Properties doParse(String data) throws IOException {
-		Properties properties = new Properties();
-		properties.load(new StringReader(data));
+		Properties properties = new OrderedProperties();
+		StringReader reader = new StringReader(data);
+		BufferedReader br = new BufferedReader(reader);
+
+		String key;
+		String value;
+		String[] sourceArr;
+		String[] valueArr;
+		for (String line = br.readLine(); line != null; line = br.readLine()) {
+			sourceArr = line.split("=");
+			if (sourceArr.length < 2) {
+				log.debug("ignore no properties format line :{}", line);
```

Comment:
There should be a bug here , missing keyword of `conitnue`  .In addition, the logging level here should be `warn` that is better

Reply:
> There should be a bug here , missing keyword of `conitnue` .In addition, the logging level here should be `warn` that is better

I thought about this situation is part of properties format wrong , but we ignore it and make other pass , that include "annotation" , if you suggest level should be `warn`,so I rewrite this code distinguish between "annotation" and "ignore"